### ISSUE TITLE
BicycleGAN on Citycscapes

### ISSUE THREAD

**Issue by SolidShen** [junyanz/BicycleGAN#67]
Hi Junyan, have you trained bicycleGAN on cityscapes(Label2Photo like pix2pix)? How does it go? Can the model also generate various realistic images from label? Since I don't see any results on cityscapes dataset from your paper and this repo. Also ,could you provide recommended network params for cityscapes? Thanks!

**Comment by junyanz** [comment]
We haven't tried it. I recommend that you use more recently developed [pix2pixHD](https://github.com/NVIDIA/pix2pixHD) and [SPADE](https://github.com/NVlabs/SPADE) architectures with BicycleGAN loss functions. They work better than the original pix2pix.

**Comment by SolidShen** [comment]
Thanks! I will try it and update the result.

**Comment by Seanseattle** [comment]
@SolidShen, hello, I have tried BicycleGAN on Cityscapes. However, the test results are too strange. 
![_frankfurt_000000_003025_leftImg8bit](https://user-images.githubusercontent.com/19276714/75845526-56646b80-5e14-11ea-8852-ad2a58ac2380.png)

 
The train results are normal:
![epoch195_fake_B_random](https://user-images.githubusercontent.com/19276714/75845495-464c8c00-5e14-11ea-9491-432796b0de09.png)

I can't figure out the problem. Do you have any idea?

**Comment by junyanz** [comment]
There might be overfitting here. Have you used cropping as part of data augmentation? Also, the cityscape data is 10x larger than datasets such as facades. So you may want to use models from early epochs.
